packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -186,6 +199,46 @@ export class InteractionTracker {
     });
   }
 
+  private interceptXHRTasks(task: AsyncTask) {
```

Comment:
Nit: usually we name things that are acronyms in title case, so this would be `interceptXhrTasks`